ci-operator/templates/openshift/installer/cluster-launch-installer-libvirt-e2e.yaml

```diff
@@ -205,11 +205,11 @@ objects:
         #!/bin/bash
         set -euo pipefail
         export PATH=/home/packer:$PATH
-        mkdir -p ${ARTIFACT_DIR}/junit
+        mkdir -p /tmp/artifacts/junit
```

Comment:
I would have preferred you fix this by adding ARTIFACT_DIR to the env, so that we remained consistent across jobs.

Reply:
This is the follow-up: https://github.com/openshift/release/pull/11313